### PR TITLE
fix: bucket only linode

### PR DESCRIPTION
**Pull request by srodenhuis**
Buckets can only be set when the OBJ provider is Linode. When using Minio, all buckets are pre-provisioned and bucket names can not be changed.

### CODE REVIEW

**Comment by j-zimnowoda** [comment]
Is this PR have corresponding apl-api PR?

File: tests/fixtures/env/settings.yaml [hunk 1]
```diff
@@ -96,9 +96,6 @@ platformBackups:
             enabled: true
             retentionPolicy: 7d
             schedule: 0 0 0 * * *
-    persistentVolumes:
```
Comment (j-zimnowoda) [inline tests/fixtures/env/settings.yaml:99]: please add migration

File: tests/fixtures/env/settings.yaml [hunk 2]
```diff
@@ -52,16 +52,16 @@ kms:
             keys: somesecretvalue
             tenantId: somesecretvalue
 obj:
-    buckets:
```
Comment (j-zimnowoda) [inline tests/fixtures/env/settings.yaml:55]: please add migrations to values-changes.yaml

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
missing values migrations

**Comment by srodenhuis** [comment]
corresponding PR for API: https://github.com/linode/apl-api/pull/546

**Comment by j-zimnowoda** [comment]
Migration goes well
image
 but there is templating issue
 ```
  otomi:cmd:template:template:info # Templating STARTED +0ms
WARNING: release knative-serving-artifacts needs knative-operator, but knative-operator is not installed due to installed: false. Either mark knative-operator as installed or remove knative-operator from knative-serving-artifacts's needs
in helmfile.d/helmfile-04.databases.yaml: failed to render values files "../values/harbor/harbor-otomi-db.gotmpl": failed to render [../values/harbor/harbor-otomi-db.gotmpl], because of template: stringTemplate:25:34: executing "stringTemplate" at <$obj.linode.buckets.cnpg>: map has no entry for key "buckets"
  otomi:global:error Error: WARNING: release knative-serving-artifacts needs knative-operator, but knative-operator is not installed due to installed: false. Either mark knative-operator as installed or remove knative-operator from knative-serving-artifacts's needs
  otomi:global:error in helmfile.d/helmfile-04.databases.yaml: failed to render values files "../values/harbor/harbor-otomi-db.gotmpl": failed to render [../values/harbor/harbor-otomi-db.gotmpl], because of template: stringTemplate:25:34: executing "stringTemplate" at <$obj.linode.buckets.cnpg>: map has no entry for key "buckets"
  otomi:global:error     at hfCore (/Users/jzimnowo/workspace/linode/apl-core/src/common/hf.ts:55:17)
  otomi:global:error     exit code: 1
  otomi:global:error     at ChildProcess.<anonymous> (/Users/jzimnowo/workspace/linode/apl-core/node_modules/zx/dist/bundle.cjs:15261:22)
  otomi:global:error     at ChildProcess.emit (node:events:517:28)
  otomi:global:error     at ChildProcess.emit (node:domain:489:12)
  otomi:global:error     at maybeClose (node:internal/child_process:1098:16)
  otomi:global:error     at Socket.<anonymous> (node:internal/child_process:450:11)
  otomi:global:error     at Socket.emit (node:events:517:28)
  otomi:global:error     at Socket.emit (node:domain:489:12)
  otomi:global:error     at Pipe.<anonymous> (node:net:350:12) +0ms
# binzx/otomi template -l name=gitea-otomi-db
Starting inspector on 127.0.0.1:4320 failed: address already in use
  otomi:common:prepareEnvironment:info Checking environment +0ms
  otomi:common:crypt:decrypt:info Skipping decryption +0ms
  otomi:cmd:template:template:info # Templating STARTED +0ms
WARNING: release knative-serving-artifacts needs knative-operator, but knative-operator is not installed due to installed: false. Either mark knative-operator as installed or remove knative-operator from knative-serving-artifacts's needs
in helmfile.d/helmfile-04.databases.yaml: failed to render values files "../values/gitea/gitea-otomi-db.gotmpl": failed to render [../values/gitea/gitea-otomi-db.gotmpl], because of template: stringTemplate:26:34: executing "stringTemplate" at <$obj.linode.buckets.cnpg>: map has no entry for key "buckets"
  otomi:global:error Error: WARNING: release knative-serving-artifacts needs knative-operator, but knative-operator is not installed due to installed: false. Either mark knative-operator as installed or remove knative-operator from knative-serving-artifacts's needs
  otomi:global:error in helmfile.d/helmfile-04.databases.yaml: failed to render values files "../values/gitea/gitea-otomi-db.gotmpl": failed to render [../values/gitea/gitea-otomi-db.gotmpl], because of template: stringTemplate:26:34: executing "stringTemplate" at <$obj.linode.buckets.cnpg>: map has no entry for key "buckets"
  otomi:global:error     at hfCore (/Users/jzimnowo/workspace/linode/apl-core/src/common/hf.ts:55:17)
  otomi:global:error     exit code: 1
  otomi:global:error     at ChildProcess.<anonymous> (/Users/jzimnowo/workspace/linode/apl-core/node_modules/zx/dist/bundle.cjs:15261:22)
  otomi:global:error     at ChildProcess.emit (node:events:517:28)
  otomi:global:error     at ChildProcess.emit (node:domain:489:12)
  otomi:global:error     at maybeClose (node:internal/child_process:1098:16)
  otomi:global:error     at Socket.<anonymous> (node:internal/child_process:450:11)
  otomi:global:error     at Socket.emit (node:events:517:28)
  otomi:global:error     at Socket.emit (node:domain:489:12)
  otomi:global:error     at Pipe.<anonymous> (node:net:350:12) +0ms
  ```

**Review by j-zimnowoda — APPROVED** [review]
fixed dev cluster manually

**Pull request merged**